### ISSUE TITLE
Unified groupId

### ISSUE THREAD

**Issue by oscarlvp** [STAMP-project/docs-forum#10]
It would be nice to have a unified groupId for the projects born under STAMP.
It could be something like `org.stamp` or `eu.stamp`.

**Comment by bbaudry** [comment]
+1

**Comment by danglotb** [comment]
Hi, 

since the STAMP-project does not own the domain `eu.stamp`, we must move  to the groupID `eu.stamp-project` to deploy on maven central.

**Comment by oscarlvp** [comment]
We checked on ovh.com and the stamp.eu domain can be bought for 5 euros. What do you think? :)

**Comment by danglotb** [comment]
I already deployed some artifacts under the groupID `eu.stamp-project` but I guess it is not a problem to deploy under a new one. 

It seems that somebody is using [`stamp.eu`](http://stamp.eu/).

**Comment by vmassol** [comment]
> I already deployed some artifacts under the groupID eu.stamp-project but I guess it is not a problem to deploy under a new one.

Could you tell me where you deployed them? I can't find them on Maven Central (https://search.maven.org/#search%7Cga%7C1%7Cdescartes) nor on the custom STAMP Maven repo at https://github.com/STAMP-project/stamp-maven-repository (there's only an old 0.1-SNAPSHOT version there).

Note: I'm ready on the xwiki project to commit. All I'm missing is a released version of descartes :)

Thx!

**Comment by CaelInria** [comment]
@danglotb : Not really, it's for sale. We checked on ovh too, and it is transferable :)
@vmassol : I'm working on it. :)

**Comment by danglotb** [comment]
Okay.

I already deployed 2 artifacts under the groupID [`eu.stamp-project`](https://search.maven.org/#search%7Cga%7C1%7Ceu.stamp-project) on maven central.

But, about the package, I used `eu.stamp.project` rather than `eu.stamp_project` but it takes only a PR to fix this.

Cheers.
